doc/api/child_process.markdown

```diff
@@ -755,7 +784,7 @@ grep.stdin.end();
 * `message` {Object}
 * `sendHandle` {Handle object}
 * `callback` {Function}
-* Return: Boolean
+* Return: Boolean | Promise
```

Comment:
@gritzko The `crypto.randomBytes` function is already dual-mode - it will act synchronously without a callback, and asynchronously with a callback. While a different situation from `send`, I'd imagine it would interfere in a similar way.